autonomous.py

```diff
@@ -333,6 +311,61 @@ def executeAuton(self):
             self.taskListCounter += 1
             
         return False
+
+
+        """ PATH_TO_NOTE with x and y adjustment, task jumping        
```

Comment:
Deleted commented out code—we can always find it in version control if we need it.